pkg/networkserver/observability.go

```diff
@@ -106,6 +106,7 @@ var (
 	evtDropDataUplink = events.Define(
```

Comment:
There's a bunch of typing left here, e.g. for MAC events - please file an issue and I will address it.

Reply:
I added data types for most (if not all) of the NS events. I did notice that the following seem to be unused:

- `evtEnqueueProprietaryMACAnswer`
- `evtEnqueueProprietaryMACRequest`
- `evtReceiveProprietaryMAC`